code/game/machinery/bots/secbot.dm

```diff
@@ -1115,6 +1122,32 @@ Auto Patrol: []"},
 	icon_state = "bsecbot0"
 	icon_initial = "bsecbot"
 
+	unsafe_weapons =list(
+		/obj/item/weapon/gun,
+		/obj/item/weapon/melee,
+		/obj/item/toy/,
+		/obj/item/ashtray,
+		/obj/item/candle,
+		/obj/item/weapon/bananapeel,
+		/obj/item/weapon/soap,
+		/obj/item/weapon/bikehorn,
+		/obj/item/weapon/wrench,
+		/obj/item/weapon/wrench/socket,
```

Comment:
If you're already checking for `wrench` this isn't needed, I think.

Reply:
good catch